lib/lint-helpers.js

```diff
@@ -49,11 +49,56 @@ exports.getLatestElectronVersion = () => {
 
 exports.fetchLatestVersion = () => {
   return Eval.execute(() => {
+    let isPositiveNumber = (x) => {
+      return /^\d+$/.test(x)
+    }
+
+    let semverComparer = (semver1, semver2) => {
+      let version1segments = semver1.split('.')
+      let version2segments = semver2.split('.')
+
+      let validateSegments = (segment) => {
+        for (var i = 0; i < segment.length; ++i) {
+          if (!isPositiveNumber(segment[i])) {
+            return false
+          }
+        }
+        return true
+      }
+
+      if (!validateSegments(version1segments) || !validateSegments(version2segments)) {
+        return NaN
+      }
+
+      for (var i = 0; i < version1segments.length; ++i) {
+        if (version2segments.length === i) {
+          return 1
+        }
+
+        if (version1segments[i] === version2segments[i]) {
+          continue
+        }
+        if (version1segments[i] > version2segments[i]) {
+          return 1
+        }
+        return -1
+      }
+
+      if (version1segments.length !== version2segments.length) {
+        return -1
+      }
+      return 0
+    }
+
     window.fetch('https://atom.io/download/atom-shell/index.json')
       .then((response) => {
         return response.json()
       }).then((versions) => {
-        window.__devtron.latestElectronVersion = versions[0].version
```

Comment:
Thanks for catching this issue, I think it might be a simpler approach to use the https://www.npmjs.com/package/semver module for something like this, it has methods to get the greater version when given two versions.